sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/Validations.scala

```diff
@@ -161,6 +167,59 @@ object Validations {
     }
   }
 
+  private def getEventType(esEntity: Class[_]): Class[_] = {
+    val genericTypeArguments = esEntity.getGenericSuperclass
+      .asInstanceOf[ParameterizedType]
+      .getActualTypeArguments
+    genericTypeArguments(1).asInstanceOf[Class[_]]
+  }
+
+  private def missingEventHandlerValidations(
+      component: Class[_],
+      updateMethodPredicate: Method => Boolean): Validation = {
+    val methods = component.getMethods.toIndexedSeq
+
+    if (hasEventSourcedEntitySubscription(component)) {
+      eventSourcedEntitySubscription(component)
+        .map { classLevel =>
+          val eventType = getEventType(classLevel.value())
+          if (!classLevel.ignoreUnknown() && eventType.isSealed) {
+            val effectMethodsInputParams: Seq[Class[_]] = methods
+              .filter(updateMethodPredicate)
+              .map(_.getParameterTypes.last) //last because it could be a view update methods with 2 params
+            val errors = eventType.getPermittedSubclasses
+              .filterNot(effectMethodsInputParams.contains)
+              .map(clazz => s"Missing event handler for ${clazz.getName}")
```

Comment:
```suggestion
              .map(clazz => s"Missing event handler for '${clazz.getName}'")
```

Not sure if we have wrapped things in single quotes in other parts. Worth to check and align eventually.

Reply:
we have, in some proxy errors but in other cases we don't, anyway I like it with single quotes